content/blog/code-to-containers.md

```diff
@@ -1,19 +1,20 @@
 ---
-title: "From Commit to Container"
-featured: false
-description: >
-    Building Containers from your code.
 date: 2020-05-29
-topics:
-- Containers
+description: 'Building Containers from your code.
```

Comment:
Is there any way to rerun the script without adding these multiple newlines to the description? Seems a bit odd

Reply:
So, before they were multiline / block strings in YAML using `>` (not sure why?) I guess this is how the YAML library translated them when writing back to the file, but.........I checked the pages and it doesn't actually seem to display / show them. So functionally no effect (afaics), but I agree it's weird.

Reply:
I guess I could re-post-process them all and remove all \n from description fields? Idk, lmk?